### PR TITLE
Space ships evenly so they don't overlap each other

### PR DESCRIPTION
**Pull request by Zeikko**

### CODE REVIEW

File: source/shift/shift.gd [hunk 1]
```diff
@@ -9,20 +9,29 @@ var ship_counter: int = 0:
...
 
 func _ready() -> void:
+	create_possible_angles()
 	create_ship()
 	security_rules.push_back(SecurityRule.create_security_rule(security_rules))
 	Ui.update_security_briefing()
 
 
+func create_possible_angles():
+	var number_of_angles: float = 16
```
Comment (Yilun-Sun) [inline source/shift/shift.gd:25]: can be `int` ?

File: source/shift/shift.gd [hunk 2]
```diff
@@ -9,20 +9,29 @@ var ship_counter: int = 0:
...
+	var angle_difference: float = 2 * PI / number_of_angles
```
Comment (Yilun-Sun) [inline source/shift/shift.gd:26]: Got a warning: The variable `angle_difference` has the same name as a built-in function. 
But I think it is fine for now. I can not came up any name better than this

**Review by Yilun-Sun — APPROVED** [review]
LGTK!

File: source/shift/shift.gd (hunk 2 above)
Comment (Zeikko) [reply source/shift/shift.gd:26]: Ah! Let's change it to `angle_diff` for example

File: source/shift/shift.gd (hunk 1 above)
Comment (Zeikko) [reply source/shift/shift.gd:25]: True!

**Pull request merged**